### PR TITLE
Changing np.int into np.int_ to follow the new protocol

### PR DESCRIPTION
**Pull request by jgolebiowski**
Changing np.int into np.int_ to follow the new protocol - https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.int_

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by mseeger** [comment]
This does not suffice, we can use #468 . Also need to remove Ray Tune touching tests

File: syne_tune/optimizer/schedulers/searchers/bore/bore.py [hunk 1]
```diff
@@ -258,9 +258,9 @@ def _train_model(self, train_data, train_targets):
             self.model = CalibratedClassifierCV(
                 self.model, cv=2, method=self.calibration
             )
-            self.model.fit(X, np.array(z, dtype=np.int))
+            self.model.fit(X, np.array(z, dtype=np.int_))
```
Comment (aaronkl) [inline syne_tune/optimizer/schedulers/searchers/bore/bore.py:261]: is this the recommend way by Numpy now? It looks a bit odd with the underscore

**Review by aaronkl — APPROVED** [review]

File: syne_tune/optimizer/schedulers/searchers/bore/bore.py (hunk 1 above)
Comment (mseeger) [reply syne_tune/optimizer/schedulers/searchers/bore/bore.py:261]: Please use the other PR I opened. We want to have np.int64 here.

Comment (jgolebiowski) [reply syne_tune/optimizer/schedulers/searchers/bore/bore.py:261]: yesh approved this. I think int_ is the updated version of np.int - I agree int64 is probably safer but didnt want to change the semantics of the code.

**Comment by jgolebiowski** [comment]
Closing since the change is already in https://github.com/awslabs/syne-tune/pull/468

**Pull request closed without merging**